### PR TITLE
Add static content for confirm company screen

### PR DESCRIPTION
**Pull request by PhilipGlover**
Language files:
- `confirm_company_company_name` looks a bit odd, but is that our convention?
- added "Confirm and continue into the global rather than the confirm-company file
- perhaps `start_main_title` should be moved to global since it's also used in the banner
- added values for the banner and header

Screenshot 2024-03-01 at 2 26 31 pm

Screenshot 2024-03-01 at 2 26 38 pm

### CODE REVIEW

File: locales/en/global.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "global_banner_new_service": "This is a new service - your ",
+    "global_banner_feedback": "feedback",
```
Comment (AlisonGriffiths) [inline locales/en/global.json:3]: Would you be ok to change this one to be "global_banner_feedback_link"

File: locales/en/global.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    "global_banner_help_improve": " will help us to improve it.",
+    "global_confirm_continue": "Confirm and continue"
```
Comment (AlisonGriffiths) [inline locales/en/global.json:5]: Then change this one to be "global_confirm_continue_button"

**Review by AlisonGriffiths — CHANGES_REQUESTED** [review]

File: locales/cy/confirm-company.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "confirm_company_main_title": "Cadarnhau mai hwn yw’r cwmni cywir",
+    "confirm_company_company_name": "Enw’r cwmni",
+    "confirm_company_company_number": "Rhif y cwmni",
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply locales/cy/confirm-company.json:4]: cool

File: src/views/router_views/confirmCompany/confirmCompany.njk [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        <h1 class="govuk-heading-l">{{ i18n.confirm_company_main_title }}</h1>
+        <form action="" method="post">
+
+            {% set notDueWarningHTML %}
+            <p style="font-weight: 400; display: inline;">
+                {{ i18n.confirmNotDueToFile }}
+                <strong>{{notDueWarning.newNextMadeUptoDate}}</strong>
+                {{ i18n.confirmReviewPeriodChange }}
+            </p>
+            {% endset %}
```
Comment (AlisonGriffiths) [inline src/views/router_views/confirmCompany/confirmCompany.njk:26]: Do we need this notDueWarningHTML section here. Can't see it on the prototype

File: locales/en/global.json (hunk 1 above)
Comment (PhilipGlover) [reply locales/en/global.json]: yep, done

File: src/views/router_views/confirmCompany/confirmCompany.njk [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+            {{ govukButton({
+          attributes: {
+            id: "submit",
+            "data-event-id": "confirm-and-continue-button"
+          },
+          text: i18n.global_confirm_continue
+        }) }}
+        </form>
+        <p class="govuk-body">
+            <a href="/appoint-update-remove-company-officer/company-number?lang={{lang}}" data-event-id="choose-a-different-company-link">{{ i18n.confirm_company_choose_different_company }}</a>
```
[the remark this answers is not in the document]
Comment (PhilipGlover) [reply src/views/router_views/confirmCompany/confirmCompany.njk]: added.

**Review by AlisonGriffiths — APPROVED** [review]

**Pull request merged**